eodag/plugins/search/qssearch.py

```diff
@@ -1281,8 +1280,8 @@ def normalize_results(
         return products
 
     def discover_queryables(
-        self, product_type: Optional[str] = None
-    ) -> Optional[Dict[str, Tuple[Annotated[Any, FieldInfo], Any]]]:
+        self, product_type: Optional[str] = None, **kwargs: Optional[Dict[str, Any]]
```

Comment:
- **kwargs type hint is incorrect, it should be Any. In kwargs we only type hint the contained elements
- kwargs is missing in docstring